Unity.gitignore

```diff
@@ -23,9 +23,15 @@
 # Autogenerated Jetbrains Rider plugin
 /[Aa]ssets/Plugins/Editor/JetBrains*
 
+# Plastic SCM meta and ache directory
+.plastic/
```

Comment:
@whisper3zzz do you believe that it is desirable to always ignore this directory? Would this hinder developers from contributing to Unity projects they source from GitHub?

Searching through public GitHub repositories, I see many projects that are including this directory. It is difficult to determine whether this is desirable in all cases, which would make this undesirable for us to include here.